pkg/auditd/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM alpine:edge AS mirror
```

Comment:
This is not reproducible, so we can't use this like this with tree hashes. The options are either to pin this to a specific sha256 hash, or to build auditd from source, or not use the standard Makefile.